src/fn_call.rs

```diff
@@ -241,24 +242,27 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
             }
 
             "__rust_maybe_catch_panic" => {
-                // fn __rust_maybe_catch_panic(f: fn(*mut u8), data: *mut u8, data_ptr: *mut usize, vtable_ptr: *mut usize) -> u32
-                // We abort on panic, so not much is going on here, but we still have to call the closure
+                // fn __rust_maybe_catch_panic(f: fn(*mut u8), data: *mut u8, data_ptr: *mut usize,
+                //                             vtable_ptr: *mut usize) -> u32
```

Comment:
this change decreases readability. Either keep the previous or move to block indent

Reply:
Fair.